private-rpc/src/permissions/authorizer.ts

```diff
@@ -45,12 +47,27 @@ export class Authorizer {
         return this.postReadFilters.get(`${address}:${method}`) || null;
     }
 
+    isAddressWhitelisted(address: Address): boolean {
```

Comment:
Does it make sense to call this something like "addressWhitelistedForExplorer"? Because this does not forbides other addresses to use the rest of the system, is just to access the explorer.